pages/index.tsx

```diff
@@ -18,7 +18,7 @@ type RolesType = {
   member: boolean;
 };
 
-type MemberType = {
+type UserType = {
```

Comment:
- can we reuse the type from `"../../types/MembersSection.type";` like done in `MembersCard/Presentation.tsx` ?

Reply:
Yes, we can. Now It's reused with the latest commit 👍🏻